docs/source/tutorial.rst

```diff
@@ -0,0 +1,167 @@
+.. tutorial
+
+Tutorials
```

Comment:
I suggest converting this into a notebook (for example docs/source/notebooks/subset.ipynb).

Reply:
@huard 
to centralise notebooks they are a notebook repository: 
https://github.com/bird-house/notebooks

Perfer to have the docs in rst format to avoid unecessary downloads. 
I included the link to the original notebook to open optionally.

Reply:
How do you make sure that the rst file is in sync with the code ? Ie that it does not become outdated ?

Reply:
@huard 
Will answer here: https://github.com/bird-house/notebooks/issues/2